lib/runtime.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * This "fake" module is just a stub to allow IDEs to resolve the `BUILDTIME_SYMBOL_TO_STRING` macro and provide
+ * autocomplete/auto-import for it in TypeScript files.
+ * NOTE: This does not influence the behavior of this plugin at all.
+ */
+declare module 'webpack-buildtime-symbols' {
+    export const BUILDTIME_SYMBOL_TO_STRING: (symbol: any) => string;
+}
```

Comment:
note: file must be a sibling of the runtime.js file or IDE will keep reporting it as a warning for missing .d.ts file